src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/services/storage/RejectedContainerCleanerTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.blobrouter.services.storage;
+
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.models.BlobItem;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.blobrouter.services.RejectedBlobChecker;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.RejectedContainerCleaner;
+import uk.gov.hmcts.reform.blobrouter.util.BlobStorageBaseTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class RejectedContainerCleanerTest extends BlobStorageBaseTest {
+
+    @Mock RejectedBlobChecker blobChecker;
+
+    @Test
+    void should_delete_files_from_rejected_container() {
+        // given
+        var normalContainer = createContainer("sample-container");
+        var rejectedContainer = createContainer("sample-container-rejected");
```

Comment:
would be nice to use actual object as it is not clear what object type is returned from createContainer ?

Reply:
True... but it's a test ;)
OK, I'll remove `var`

Reply:
done